cloudevents/spec.md

```diff
@@ -165,7 +165,7 @@ schema for its payload.
           "schemaformat": "Protobuf/3.0",
           "schema": "syntax = \"proto3\"; message Metrics { float metric = 1; } }"
 
-          "metaurl": "https://example.com/endpoints/com.example.telemetry/messages/com.example.telemetry/meta",
+          "metaurl": "https://example.com/endpoints/com.example.telemetry/messages/com.example.telemetry/metadata",
```

Comment:
I'm intrigued as to this change, given that we're not changing (here, anyway) "meta" to "metadata" other than in this URL.

It's as if the spec can't decide whether "metadata" is clearer than "meta" or not...

Reply:
No big mystery... I messed up :-)
For some reason while doing $json->$structure I also made the meta->metadata change even though we agreed not to. By the time I realized my mistake I had already made lots of other edits and rather than starting over I did a metadata->meta round of edits... and missed some. Good catch!